navigation/ca_hector/launch/geotiff_mapping.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+
+<!-- Helper launchfile to store the map created with Hector SLAM in
+tif format.
+To save the map at any point of execution, you need to run:
+
+rostopic pub <ns>/syscommand std_msgs/String "savegeotiff"
```

Comment:
You could create a launch file to do this automatically: https://answers.ros.org/question/216036/rostopic-node-in-launch-files/?answer=216044#post-id-216044

And call the launchfile that will handle the namespace automatically.

Reply:
Sure, thanks!